how-to/security/openssh-server.md

```diff
@@ -65,35 +67,57 @@ Many other configuration directives for `sshd` are available to change the serve
 
 ## SSH keys
 
-SSH allows authentication between two hosts without the need of a password. SSH key authentication uses a **private key** and a **public key**.
+SSH allows authentication between two hosts without the need of a password, using cryptographic keys instead.
+{term}`SSH-key` authentication uses a **private key** and a **public key**.
 
-To generate the keys, run the following command:
+To generate a key pair, run the following command:
```

Comment:
Maybe just expand this sentence a bit, to say which type of algorithm will be used. Suggestion:
`To generate a key pair using the Ed25519 Elliptic Curve algorithm, run the following command:`

Then down below, perhaps instead of saying "If needed", just say "For an RSA key, do ...". If there are good resources out there explaining at a high level the differences, that could also be linked here.